src/sentry/static/sentry/app/components/customResolutionModal.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import jQuery from 'jquery';
+import Modal from 'react-bootstrap/lib/Modal';
+import underscore from 'underscore';
+
+import TimeSince from './timeSince';
+import Version from './version';
+
+import {Select2FieldAutocomplete} from './forms';
+import {t} from '../locale';
+
+export default React.createClass({
+  propTypes: {
+    onSelected: React.PropTypes.func,
+    onCanceled: React.PropTypes.func,
+    orgId: React.PropTypes.string.isRequired,
+    projectId: React.PropTypes.string.isRequired,
+    show: React.PropTypes.bool
+  },
+
+  getInitialState() {
+    return {version: ''};
+  },
+
+  componentDidUpdate(prevProps, prevState) {
+    if (this.props.show) {
+      // XXX(cramer): this is incorrect but idgaf
+      jQuery('.modal').attr('tabindex', null);
+    }
+  },
+
+  onSubmit() {
```

Comment:
Aside/musing/ignore this – in the future, we should having a naming policy that distinguishes between a prop name and a prop handler.

Something like `onX` for prop names/event names, `handleX` for the prop/event handler:

```javascript
<button onCancel={this.handleCancel}/>
```